package.json

```diff
@@ -14,16 +14,18 @@
     "build": "ember build",
     "lint:js": "eslint ./*.js app blueprints config lib server tests",
     "start": "ember serve",
-    "test": "ember test"
+    "test": "COVERAGE=true ember test"
   },
   "devDependencies": {
     "active-model-adapter": "^2.1.1",
     "broccoli-asset-rev": "^2.4.5",
+    "codecov": "^3.0.2",
```

Comment:
The exact version and its dependencies is not tagged in yarn.lock. Should be tagged in my opinion. It may cause build error while new version releases with matching semver pattern and it introduce incompatibility in that release.